### PR TITLE
Update previous Crystal release - 1.3.2

### PR DESCRIPTION
**Pull request by straight-shoota**

### CODE REVIEW

**Review by beta-ziliani — APPROVED** [review]
🚀

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        crystal_bootstrap_version: [1.2.2]
+        crystal_bootstrap_version: [1.2.2, 1.3.0, 1.3.1]
```
Comment (lbguilherme) [inline .github/workflows/linux.yml:18]: Is it necessary to test against both 1.3.0 and 1.3.1?

Comment (Sija) [reply .github/workflows/linux.yml:18]: Testing against `1.3.1` should suffice.

Comment (straight-shoota) [reply .github/workflows/linux.yml:18]: Ha, that's a good point. #11716 needs improvement then ;)

Comment (straight-shoota) [reply .github/workflows/linux.yml]: [`44dee47` (#11716)](https://github.com/crystal-lang/crystal/pull/11716/commits/44dee477147c224f962b80cf09010ea44866250e)

File: scripts/release-update.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env sh
```
Comment (beta-ziliani) [inline scripts/release-update.sh]: why is this file here? it's a duplicate of #11716 right?

Comment (straight-shoota) [reply scripts/release-update.sh]: Yes, this was wrongly commited here. I removed it in the last commit aba20e4, it's not part of the merge diff.

File: shell.nix [hunk 3]
```diff
@@ -52,13 +52,13 @@ let
   # Hashes obtained using `nix-prefetch-url --unpack <url>`
   latestCrystalBinary = genericBinary ({
     x86_64-darwin = {
-      url = "https://github.com/crystal-lang/crystal/releases/download/1.3.1/crystal-1.3.1-1-darwin-universal.tar.gz";
-      sha256 = "sha256:18cfdp4k2cfnawxgg8k2pzy0657v5cl7sj8kyz76va5ajhrdws66";
+      url = "https://github.com/crystal-lang/crystal/releases/download/1.3.2/crystal-1.3.2-1-darwin-universal.tar.gz";
+      sha256 = "sha256:1jxc5c82wxigza2z0ck51dnwpvfzba0gsczrjxgfjysqn6ki7v3q";
```
[the remark this answers is not in the document]
Comment (straight-shoota) [reply shell.nix:56]: Yes, nix does something differently here. I'm not sure what exactly, maybe it takes the file name into account or something.
The hashes are correct for nix. They are calculated via `nix-prefetch-url` as mentioned in the comment above this piece of code.

[the remark this answers is not in the document]
Comment (beta-ziliani) [reply shell.nix:56]: 🙈 thanks!

**Review by beta-ziliani — APPROVED** [review]
🚀
